tests/BUILD.bazel

```diff
@@ -1,10 +1,20 @@
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
```

Comment:
Let's exclude the `//tests` directory and not make changes here. None of the `.bzl` files here should be imported by other repos, and almost nothing here follows Gazelle conventions.

Reply:
Done